perceval/backends/kitsune.py

```diff
@@ -107,8 +109,28 @@ def fetch(self, offset=DEFAULT_OFFSET):
         drop_questions = offset - page_offset
         current_offset = offset
 
-        for raw_questions in self.client.get_questions(offset):
+        questions_page = self.client.get_questions(offset)
+
+        while True:
+            try:
+                raw_questions = next(questions_page)
+            except StopIteration:
+                break
+            except requests.exceptions.HTTPError as e:
+                # Continue with the next page if it is a 500 error
+                if e.response.status_code == 500:
+                    traceback.print_exc()
+                    logging.error("Problem getting Kitsune questions. Loosing %i questions. Going to the next page.", KitsuneClient.ITEMS_PER_PAGE)
```

Comment:
Use `logger` object, not `logging`. You also have to split the line. It is too long.